content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,33 @@
+====================
+Electronic invoicing
+====================
+
+EDI, or electronic data interchange, allows to rapidly send information digitally using standard
+formats.
+
+This feature enables automating the administration between companies and might also be required by
+some governments for fiscal control or to facilitate the administration.
+
```

Comment:
Add an explanation that there are various formats (it is still an explanation, so no need to link this info to what Odoo does).